gitprompt

```diff
@@ -22,7 +22,7 @@ function parse_git_branch {
   git rev-parse --git-dir &> /dev/null
 
   git_status="$(git status 2> /dev/null)"
-  branch_pattern="^On branch ([-_A-Za-z0-9\/]*)[^A-Za-z0-9]"
+  branch_pattern="^On branch ([-_A-Za-z0-9\/\.]*)[^A-Za-z0-9]"
```

Comment:
What kind of monster uses dots in a branch name?

Reply:
myself